graphql_subscription_manager/__init__.py

```diff
@@ -132,8 +132,19 @@ async def retry(self):
         _LOGGER.debug("Close websocket")
         await self._close_websocket()
         _LOGGER.debug("Restart")
-        self._retry_timer = self.loop.call_soon(self.start)
-        _LOGGER.debug("Reconnecting to server.")
+        self._retry_count += 1
+
+        # Jitter of 1 to 60 seconds
+        jitter = random.randrange(1, 60)
+
+        # Exponential backoff
+        backoff = pow(self._retry_count, 2)
```

Comment:
Having several thousand devices reconnect over 2 minute period would still lead to X thousand requests per minute all having to do authenication against the backend. Spreading that out would reduce risk of services crashing. Do you think that would cause big inconveniece from HA perspective?

Reply:
2 minutes without data is pretty bad if you do fuse load balancing. 
I am ok with the change if it is required to use the Tibber API.

Reply:
It's not required, I'm probably a bit overly cautious (though you should note that fuses usually do sustain longer period of overload before blowing).

How about we make it so that first retry occurs at random within 1 - 60 seconds?

Reply:
@ztamas83 you could adjust this in your PR #34?

Reply:
I've now adjusted this a bit.